src/components/vuestic-components/va-checkbox/VaCheckbox.vue

```diff
@@ -128,6 +129,12 @@ export default {
       },
     },
 
+    autofocus: {
```

Comment:
I'm not seeing too much value for autofocus being contextable.

Reply:
Yeap. Now it's an unnecessary prop